protocol/communities/manager.go

```diff
@@ -980,7 +980,13 @@ func (m *Manager) RemovePrivateKey(id types.HexBytes) (*Community, error) {
 	return community, nil
 }
 
-func (m *Manager) ExportCommunity(id types.HexBytes) (*ecdsa.PrivateKey, error) {
+type CommunityImportInfo struct {
+	Shard        *common.Shard   `json:"shard"`
+	TopicKey     *types.HexBytes `json:"protectedTopicPrivateKey,omitempty"`
```

Comment:
`TopicKey` could be `json:"topicKey"` and `CommunityKey` could be `json:"communityKey"`? Or is there a reason to not apply this convention?